functions/path_join.pp

```diff
@@ -3,10 +3,12 @@
 # will format a windows path into a equilivent unix like path.  This type of unix like
 # path will work on windows.
 #
-# @param dirs [Array[String]] - Joins two or more directories by file separator.
-# @return [Stdlib::Absolutepath] - The joined path
-# @example extlib::path_join('/tmp', 'test', 'libs') returns '/tmp/test/libs'
-# @example extlib::path_join('c:', 'test', 'libs') returns '/c/test/libs'
+# @param dirs Joins two or more directories by file separator.
+# @return [Stdlib::Absolutepath] The joined path
```

Comment:
IMO, `[Stdlib::Absolutepath]` shouldn't be needed here, but puppet-strings explodes if you remove it...

Reply:
https://tickets.puppetlabs.com/projects/PDOC/issues/PDOC-260